python/src/MaximumLikelihoodFactory_doc.i.in

```diff
@@ -148,3 +148,45 @@ Returns
 -------
 indices : :class:`~openturns.Indices`
     Indices of fixed parameters."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::MaximumLikelihoodFactory::BuildGaussianEstimator
+"Compute distribution of parameters.
+
+Compute the asymptotic distribution of parameters for a regular distribution via the delta method.
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    Inferred distribution
+sample : 2-d sequence of float
+    Data to infer
+
+Returns
+-------
+distribution : :class:`~openturns.Distribution`
+    Asymptotic gaussian distribution of the parameters."
```

Comment:
We are using the convention that "Gaussian" is applied to vectors and "normal" to distributions. Also, adjectives derived from proper nouns like"Gaussian" have an uppercase first letter in English.